README.md

```diff
@@ -1,11 +1,14 @@
 # prodmgr
 
-`prodmgr` is the CLI for Shasta uninstall and downgrade. It works by running
-product-specific install utility container images.
+`prodmgr` is the CLI for Shasta activation or deletion of versions of products. It works by running
+product-specific install utility container images to activate products and a generic deletion container
+to delete them.
```

Comment:
Perhaps a nitpick, but I'd put the generic deletion container mention first to emphasize that products do not need to provide their own container image to support deletion and that it only falls back on product-specific container images in order to maintain support for the `activate` functionality during a deprecation period.

Reply:
Done.